packages/portfolio-contract/src/pos-usdn-shapes.ts

```diff
@@ -0,0 +1,7 @@
+import type { TypedPattern } from '@agoric/internal';
+
+// #region Proposal Shapes
+
+export const makeUSDNGiveFields = (
+  usdcAmountShape: TypedPattern<Brand<'nat'>>,
```

Comment:
Seems weird to me.  Shouldn't this be:
```suggestion
  usdcAmountShape: TypedPattern<Amount<'nat'>>,
```

Reply:
that file is gone, right?

Reply:
Yeah, you changed things faster than I could review them. 😛